test/e2e/specs/editor/various/rich-text.spec.js

```diff
@@ -0,0 +1,615 @@
+		await editor.clickBlockToolbarButton( 'More' );
+
+		const button = page.locator( 'text=Highlight' );
+
+		// Clicks may fail if the button is out of view. Assure it is before click.
+		await button.evaluate( ( element ) => element.scrollIntoView() );
+		await button.click();
+
+		// Tab to the "Text" tab.
+		await page.keyboard.press( 'Tab' );
+		// Tab to black.
+		await page.keyboard.press( 'Tab' );
+		// Select color other than black.
+		await page.keyboard.press( 'Tab' );
+		await page.keyboard.press( 'Tab' );
+		await page.keyboard.press( 'Enter' );
+
+		expect( await editor.getEditedPostContent() ).toMatchSnapshot();
+
+		// Dismiss color picker popover.
+		await page.keyboard.press( 'Escape' );
+
+		// Navigate to the block.
+		await page.keyboard.press( 'Tab' );
+
+		// Copy the colored text.
+		await pageUtils.pressKeyWithModifier( 'primary', 'c' );
+
+		// Collapse the selection to the end.
+		await page.keyboard.press( 'ArrowRight' );
+
+		// Create a new paragraph.
+		await page.keyboard.press( 'Enter' );
+
+		// Paste the colored text.
+		await pageUtils.pressKeyWithModifier( 'primary', 'v' );
+
+		expect( await editor.getEditedPostContent() ).toMatchSnapshot();
+	} );
+
+	test( 'should paste paragraph contents into list', async ( {
+		editor,
+		page,
+		pageUtils,
+	} ) => {
+		await page.keyboard.press( 'Enter' );
+
+		// Create two lines of text in a paragraph.
+		await page.keyboard.type( '1' );
+		await pageUtils.pressKeyWithModifier( 'shift', 'Enter' );
+		await page.keyboard.type( '2' );
+
+		// Select all and copy.
+		await pageUtils.pressKeyWithModifier( 'primary', 'a' );
+		await pageUtils.pressKeyWithModifier( 'primary', 'c' );
+
+		// Collapse the selection to the end.
+		await page.keyboard.press( 'ArrowRight' );
+
+		// Create a list.
+		await page.keyboard.press( 'Enter' );
+		await page.keyboard.type( '* ' );
+
+		// Paste paragraph contents.
+		await pageUtils.pressKeyWithModifier( 'primary', 'v' );
+
+		expect( await editor.getEditedPostContent() ).toMatchSnapshot();
+	} );
+
+	test( 'should paste list contents into paragraph', async ( {
+		editor,
+		page,
+		pageUtils,
+	} ) => {
+		await page.keyboard.press( 'Enter' );
+
+		// Create an indented list of two lines.
+		await page.keyboard.type( '* 1' );
+		await page.keyboard.press( 'Enter' );
+		await page.keyboard.type( ' 2' );
+
+		// Select all and copy.
+		await pageUtils.pressKeyWithModifier( 'primary', 'a' );
+		await pageUtils.pressKeyWithModifier( 'primary', 'c' );
+
+		// Collapse the selection to the end.
+		await page.keyboard.press( 'ArrowRight' );
+
+		// Create a paragraph.
+		await page.keyboard.press( 'Enter' );
+		await page.keyboard.press( 'Enter' );
+
+		// Paste paragraph contents.
+		await pageUtils.pressKeyWithModifier( 'primary', 'v' );
+
+		expect( await editor.getEditedPostContent() ).toMatchSnapshot();
+	} );
+
+	test( 'should navigate arround emoji', async ( { editor, page } ) => {
```

Comment:
```suggestion
	test( 'should navigate around emoji', async ( { editor, page } ) => {
```

(will have to update snapshots after this change too.